tests/src/test/java/org/teavm/vm/RecordTest.java

```diff
@@ -44,6 +44,12 @@ public void toStringMethod() {
 
         int index = 0;
 
+        // assertEquals("B[x=2, y=q, z=3]", s);
```

Comment:
I could not find any spec regarding Record.toString implementation, so I guess it may vary between JDK implementations and versions. This makes it impossible to test toString implementation this way.